index.html

```diff
@@ -149,9 +135,63 @@
                         <p>Work toward your own service enterprise.</p>            
                     </div>
                     
-                </section> <!-- end myIconRow -->
+                </section> <!-- end outside-in-iconRow -->
 
 
+                <section class="row outside-in-infoRow1 outside-in-rowPadding">
+                
+                    <div class="small-12 column">
+                        <h3>For organizations that need help with accessibility</h3>
+                    </div>
+                    
+                    <div class="small-12 medium-6 column">
+                        <p>Outside-In takes your inaccessible documents and audiovisual media and makes them accessible. We use your documents to provide training and work to young people with episodic disabilities and provide you with a quality-assured accessible file. We also provide website accessibility reports and can help you understand any changes you should make to your website to make it accessible and compliant with legislation such as the Accessibility for Ontarians with Disabilities Act. If you would like to work with Outside-In to meet your accessible media needs, please contact the project coordinator to discuss your accessibility needs and rates.</p>
```

Comment:
Elsewhere in the page, the "in" part is all-caps i.e. "Outside-IN." We should try to be consistent.